rosdep/python.yaml

```diff
@@ -8580,6 +8588,10 @@ python3-systemd:
   debian: [python3-systemd]
   nixos: [python3Packages.systemd]
   ubuntu: [python3-systemd]
+python3-tables-pip:
```

Comment:
This is packaged natively for example Ubuntu https://packages.ubuntu.com/jammy/python3-tables

Please update this to use the native packages on as many platforms as you can find, as per our contributing guide.